hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOzoneManagerHASnapshot.java

```diff
@@ -42,23 +51,44 @@
  * Tests snapshot in OM HA setup.
  */
 @Timeout(300)
-@Disabled("HDDS-8645")
-public class TestOzoneManagerHASnapshot extends TestOzoneManagerHA {
+public class TestOzoneManagerHASnapshot {
+  private static MiniOzoneHAClusterImpl cluster;
+  private static OzoneClient client;
+  private static String volumeName;
+  private static String bucketName;
+  private static ObjectStore store;
+  private static OzoneBucket ozoneBucket;
+
+  @BeforeAll
```

Comment:
Please add an `@AfterAll` method to close client and shutdown the cluster.  I think that will solve the out of space problem. ;)

Reply:
Thanks for the catch. Fixed it.